sql/tsql/TSqlParser.g4

```diff
@@ -3078,6 +3078,32 @@ fetch_cursor
     : FETCH ((NEXT | PRIOR | FIRST | LAST | (ABSOLUTE | RELATIVE) expression)? FROM)?
       GLOBAL? cursor_name (INTO LOCAL_ID (',' LOCAL_ID)*)? ';'?
     ;
+    
+special_list
+    : ANSI_NULLS 
+    | QUOTED_IDENTIFIER 
+    | ANSI_PADDING 
+    | ANSI_WARNINGS 
+    | ANSI_DEFAULTS 
+    | ANSI_NULL_DFLT_OFF 
+    | ANSI_NULL_DFLT_ON 
+    | ARITHABORT 
+    | ARITHIGNORE 
+    | CONCAT_NULL_YIELDS_NULL 
+    | CURSOR_CLOSE_ON_COMMIT 
+    | FMTONLY 
+    | FORCEPLAN 
+    | IMPLICIT_TRANSACTIONS 
+    | NOCOUNT 
+    | NOEXEol 
+    | NUMERIC_ROUNDABORT 
+    | PARSEONLY 
+    | REMOTE_PROC_TRANSACTIONS 
+    | SHOWPLAN_ALL 
+    | SHOWPLAN_TEXT 
+    | SHOWPLAN_XML 
+    | XACT_ABORT
+    ;
```

Comment:
It's always better to place definitions after usings if possible (similar to methods in source code). Exceptions are recursive definitions where it's not possible.

Reply:
Do you mean after set_special at line 3084?

Reply:
Yes.